lib/Component.spec.lua

```diff
@@ -456,17 +454,12 @@ return function()
 				})
 			end
 
-			local testElement = createElement(TestComponent)
-
-			local handle = Reconciler.mount(testElement)
-
-			expect(renderCount).to.equal(1)
-
-			forceUpdate()
-
-			expect(renderCount).to.equal(2)
+			local element = createElement(TestComponent)
 
-			Reconciler.unmount(handle)
+			expect(function()
+				Reconciler.mount(element)
+				forceUpdate()
+			end).to.throw()
```

Comment:
This test will also pass if we rename `_forceUpdate`, an internal API. Can we refactor this test to use `Reconciler.reconcile` or whatnot instead?

Reply:
Good point, I'll make that change.